dali/pipeline/executor/executor.cc

```diff
@@ -706,7 +706,7 @@ void Executor::SetupOutputQueuesForGraph() {
     }
 
     // Create the buffer and events
-    if (tensor_meta.is_cpu) {
+    if (tensor_meta.storage_device == DALITensorDevice::CPU) {
```

Comment:
`DALITensorDevice` - isn't it a bit broader? Somthing like `MemoryKind`, `AddressSpace` or `StorageDevice` comes to mind.

Reply:
Done in #598